python/shark_turbine/dynamo/importer.py

```diff
@@ -276,34 +309,54 @@ def integer_attr(self, value: int, bits: int) -> MlirAttribute:
         c = self._c
         return IntegerAttr.get(IntegerType.get_signless(bits, c), value)
 
+    """Strips symbolic elements from a torch.Size object and returns shape asm"""
+    def parse_shape(self, shape: torch.Size) -> str:
+        return ",".join('?' if is_symbolic(d) else str(d) for d in list(shape))
+
+    """Return MlirType for !torch.vtensor with the given shape and dtype"""
+    def get_vtensor_type(self, shape: torch.Size, dtype: torch.dtype):
+        shape_asm = self.parse_shape(shape)
```

Comment:
Let's rename this: I wouldn't expect "parse" to apply to this situation. `format_asm_shape` reads a bit better to me.